packages/osfamilymap.yaml

```diff
@@ -1,5 +1,11 @@
 # -*- coding: utf-8 -*-
 # vim: ft=yaml
+Suse:
+  pips:
+    required:
+      pkgs:
+        - python2-pip
```

Comment:
should be python-pip, which will depend on python2-pip or python3-pip based on suse version.

Reply:
Hi @aboe76 I believe you are correct - when I tried `python-pip` salt was throwing error.  
However I'm suspect the vagrant image was not ideal - some funny issues.
So I'm downloading different suse image and report back.

Reply:
This is silly!  So python-pip will not work??
```
          ID: pip_req_pkgs
    Function: pkg.installed
      Result: False
     Comment: An error was encountered while installing package(s): Zypper command failure: Package 'python-pip' not found.
     Started: 09:19:04.574598
    Duration: 1556.153 ms
     Changes:


opensuse-leap-x64:/home/vagrant/opensds-installer/salt # zypper in python-pip
Loading repository data...
Reading installed packages...
'python-pip' not found in package names. Trying capabilities.
'python2-pip' providing 'python-pip' is already installed.
Resolving package dependencies...

Nothing to do.
```